app/furniture/marketplace/order_received_mailer/notification.html.erb

```diff
@@ -10,3 +10,4 @@
 <p>Total: <%= humanized_money_with_symbol(order.price_total) %></p>
 <p>Deliver To: <%= order.delivery_address %>
 <p>Contact Email: <%= order.contact_email %>
+<p>Contact Phone Number: <%= order.contact_phone_number %>
```

Comment:
BUG: `Contact Phone Number:` fails to render in the mailer. `order.contact_phone_number` is always nil.

Is the `cart.contact_phone_number` supposed to be passed to the `order.contact_phone_number`?

Reply:
I would recommend tracing:
- Down through the application layer to the storage layer to determine where both `Cart#contact_phone_number` and `Order#contact_phone_number` is stored , and
- Along the interaction layer to what `Controller#action` would be responsible for passing data from `Cart` to `Order` 

My current guess is that `Order.contact_phone_number` may not have the lockbox settings?

Reply:
Nah, looks like lockbox is set correctly; Have you updated the `OrderReceivedMailerPreview` to include a contact_phone_number? It is likely that the `Preview` calls the `factory`, so you'll want to search for where we define the `marketplace_order` factory and add a `Faker` call for the phone_number...